feeds/wlan-ap/interAPcomm/src/src/test/interapcommtest.c

```diff
@@ -22,34 +25,37 @@ int recv_process(void *data) {
 
 int main (int argc, char *argv[ ])
 {
-	unsigned int send = atoi(argv[1]);   /* First arg: broadcast port */
-	unsigned short port = 50000;
+	unsigned int send = atoi(argv[1]);
+	unsigned short port = 50020;
 //	char *dst_ip = "255.255.255.255";
-	char *dst_ip = "192.168.9.255";
+	char *dst_ip = "10.42.0.255";
 //	char *data = "InterAP Hello";
 	struct my_data data;
 	data.x = 1001;
 	data.y = 'H';
 	data.z = 3003;
 
-//	callback cb = recv_process;
-
-
-	printf("arg1 = %d\n", send);
+	printf("send = %d\n", send);
```

Comment:
Where do all these print statements end up?  Do we need these?

Reply:
This is only for testing, and do not get copied to the AP.